container_api/src/interface.rs

```diff
@@ -0,0 +1,82 @@
+use holochain_core_types::{error::HolochainError, json::JsonString};
+use Holochain;
+
+use jsonrpc::{jsonrpc_error, jsonrpc_success, JsonRpcRequest};
+use serde_json::{self, Value};
+use std::{
+    collections::HashMap,
+    convert::TryFrom,
+    sync::{Arc, Mutex},
+    thread,
+};
+use ws::{self, Message, Result as WsResult};
+
+use config::{Configuration, InstanceConfiguration};
+
+pub type InterfaceError = String;
+pub type InstanceMap = HashMap<String, Arc<Mutex<Holochain>>>;
+
+pub trait DispatchRpc {
+    fn dispatch_rpc(&self, rpc: &JsonRpcRequest) -> JsonString;
+}
+
+pub struct RpcDispatcher {
```

Comment:
I think I would call this thing `ContainerAPI` and move it in its own file. The implementation of `dispatch_rpc` below will grow over time to be the central point of this whole crate, no?

Reply:
I would call the set of available JSON-RPC functions Container API.